include/ada/checkers.h

```diff
@@ -7,9 +7,35 @@
 
 namespace ada::checkers {
 
-  bool ends_in_a_number(std::string_view input) noexcept;
-  bool is_windows_drive_letter(std::string_view input) noexcept;
-  bool is_normalized_windows_drive_letter(std::string_view input) noexcept;
+  // If we wish to separate definitions and declarations, we can do
+  // it with a -inl.h files.
```

Comment:
I see that this is a common C++ practice. We can follow it.

Reply:
That's how Google does it, yeah.